scss/main.scss

```diff
@@ -31,57 +45,199 @@ a {
     }
 }
 
+.form-control {
+    background: $secondaryColor;
+    border: none;
+    color: $light;
+    @include box-shadow(none)
+}
+
 #header {
     background: $primaryColor;
     padding: 20px;
-    margin-bottom: 40px;
+    margin-bottom: 20px;
 
-    .form-control {
-        background: $secondaryColor;
-        border: none;
-        color: $light;
-        @include box-shadow(none)
+    .logo {
+        font-size: 2em;
+        color: $secondaryColor;
+        font-weight: bold;
+        text-transform: uppercase;
     }
 
     .google_login{
         background-color: rgb(220, 74, 56);
         text-align: center;
         cursor: pointer;
+        color: #fff;
+        font-size: 16px;
+        padding: 5px;
     }
     
 }
 
+.h2-heading {
+    text-align: center;
+    padding: 10px 0;
+    margin: 0 0 10px 0;
+    border-bottom: 1px dotted $primaryColor;
+    color: $secondaryColor;
+}
+
+.urlLink {
+    width: auto;
+    vertical-align: top;
+}
+
+.layoutDefs {
+    h3 {
+        margin: 0px;
+        padding: 0px;
+        border-bottom: 1px dotted $primaryColor;
+        margin: 0 0 5px 0;
+    }
+    
+    a {
+        color: $primaryColor * -5;
+        vertical-align: middle;
+        &:hover {
+            color: $primaryColor;
+        }
+    }
+}
+
+.loading {
+    text-align: center;
+    font-size: 15px;
+    padding: 2em 0 0 0;
+}
+
 #sources {
     h2 {
-        text-align: center;
-        margin: 20px 0;
+        @extend .h2-heading;
     }
-
+    div.nrc:nth-child(3n+4) {
+        clear: both;
+    }
+    
 }
 
 .source {
     display: block;
     min-height: 20px;
-    line-height: 1;
     margin: 0 0 10px 0;
 
     .icon {
         @extend .inline-block;
+        width: 25px;
         font-size: 20px;
         vertical-align: middle;
+        float: left;
     }
 
-    .source-link {
-        @extend .inline-block;
+    .link {
+        @extend .urlLink;
+        padding-left: 30px;
     }
 
-    h3 {
+    @extend .layoutDefs;
+
+    a{
+        font-size: 16px;
+    }
+
+}
+
+#articles {
+    .article {
+        margin: 0 0 10px 0;
```

Comment:
It's a good practice to alphabetize your css declarations. so in this case.. border-bottom should come first then margin then padding.. i.e 
```.article { border-bottom, margin, padding }```